app/riot/src/client/preview/render-riot.js

```diff
@@ -49,7 +50,13 @@ function renderStringified({
     eval(getRidOfRiotNoise(`${compiled}`)); // eslint-disable-line no-eval
   });
   const sourceCode = `<root>${template}</root>`;
-  if (template !== '<root/>') eval(getRidOfRiotNoise(`${compiler.compile(sourceCode, {})}`)); // eslint-disable-line no-eval
+  const compiledRootSource = !tagConstructor
```

Comment:
Can you please extract this to a separate function? I think riot rendering in storybook is kinda complicated (comparing to other apps), so having it as clear and granular as possible will be helpful =)

Reply:
okay. This evening after my workday.

Reply:
refactored